test/helpers/copy_directory.dart

```diff
@@ -0,0 +1,18 @@
+import 'dart:io';
+
+import 'package:path/path.dart' as path;
+
+Future<void> copyDirectory(Directory from, Directory to) async {
```

Comment:
minor but we could also use https://pub.dev/documentation/io/latest/io/copyPath.html